frontend/src/buttons.scss

```diff
@@ -1,5 +1,38 @@
 @import './colors.scss';
 
+.svg {
```

Comment:
SVG buttons with basic styling.  Not a fan of doing it this way, but it's taking me way too much time to literally add colours to the page...